webpack.common.cjs

```diff
@@ -18,6 +18,7 @@ module.exports = {
     http: { commonjs: 'http' },
     https: { commonjs: 'https' },
     deepmerge: { commonjs2: 'deepmerge' },
-    'base64-arraybuffer': { commonjs2: 'base64-arraybuffer' }
+    'base64-arraybuffer': { commonjs2: 'base64-arraybuffer' },
+    '@sentienttechnologies/hashing': { commonjs2: '@sentienttechnologies/hashing' }
```

Comment:
I'm not sure if this is necessary. You could make the argument that the hashing library could be an external dependency in the SDK. But then the downstream asset-manager package will need to embed `hashing` within its bundle.

Reply:
So, I followed the existing pattern for production dependencies.

Reply:
The pattern here is so that the dependencies needed when running under a Node runtime rather than the browser are not bundled into the main file of the package. But the hashing library probably should be bundled into package.